scripts/dummy_placeholder.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+`use strict`;
+
+/**
+ Dummy test to show placeholder {rand}. It simulates simulate random delay based on the value.
+ */
+const lr = require('../index.js')();
+const args = require('optimist');
```

Comment:
This might need to be changed, since the dependency on optimist has been removed.